packages/search-ui-elasticsearch-connector/src/handlers/autocomplete/index.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  AutocompleteQuery,
+  AutocompleteResponseState,
+  RequestState
+} from "@elastic/search-ui";
+import resultsHandler from "./results";
+import suggestionsHandler from "./suggestions";
+
+export default async function handleRequest(
+  state: RequestState,
+  queryConfig: AutocompleteQuery,
+  host: string,
+  index: string,
+  apiKey: string,
+  queryFields: string[]
+): Promise<AutocompleteResponseState> {
+  const requests = [];
+
+  if (queryConfig.results) {
+    requests.push(
+      resultsHandler(state, queryConfig, host, index, apiKey, queryFields)
+    );
+  }
+
+  if (queryConfig.suggestions && queryConfig.suggestions.types) {
+    const promises = Object.keys(queryConfig.suggestions.types).map((type) => {
```

Comment:
as searchkit doesn't support suggestions autocomplete, the handler will return an empty array for now. Will be addressed in another issue.